src/styles/styles.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sourceRoot":"","sources":["base.scss","movie.scss"],"names":[],"mappings":"AAAA;EACI;;;AAGJ;EACI;EACA;EACA;;;ACLA;EACI;EACA;EACA;;AAGJ;EACI;EACA;EACA;EACA;EACA;;AAGJ;EACI;;AAGJ;EACI;EACA;EACA;EACA;;AAGJ;EAKI;EACA;;AAGJ;EACI;EACA;EACA;EACA;;AAGJ;EACI;EACA;EACA;;AAGJ;EAEI;EACA;;AAGJ;EACI;EACA;EACA;EACA;EACA","file":"styles.css"}
```

Comment:
I think you use some compiler for scss. It's not necessary and you can add files like this to `.gitignore`
```
src/*/**.css.map
```